src/components/TaskList.tsx

```diff
@@ -0,0 +1,25 @@
+import { useTasksStore } from "../stores/tasks";
+import { Task, TasksStore } from "../types/Task";
+import TaskCard from "./TaskCard";
+
+
+export default function TaskList() {
+  const tasks = useTasksStore((state: TasksStore) => state.tasks) as Task[];
```

Comment:
Why is the cast needed?

Reply:
You already defined the types in the zustand store

Reply:
I forgot to remove it, this is no longer needed